Makefile

```diff
@@ -565,7 +565,7 @@ style : style-c style-py
 
 CLANG_TIDY ?= clang-tidy
 %.c.tidy : %.c
-	$(CLANG_TIDY) $(TIDY_OPTS) $^ -- $(CPPFLAGS) --std=c99
+	$(CLANG_TIDY) $(TIDY_OPTS) $^ -- $(CPPFLAGS) --std=c99 -I$(CUDA_DIR)/include
```

Comment:
When `CUDA_DIR` is not defined, this becomes `-I/include` which is perfectly valid. It works fine in Travis and locally.